socorro/schemas/README.md

```diff
@@ -1,5 +1,46 @@
 # Socorro JSON Schemas
 
-This folder contains JSON Schema files describing the documents that Socorro generates. At the moment only the Processed Crash (the one generated by Socorro's processors) is described.
+This folder contains JSON Schema files describing the documents that Socorro
+generates. At the moment only the Processed Crash (the one generated by
+Socorro's processors) is described.
 
-These files will be used as a "contract" between Socorro and other systems to which we might send our data.
+These files will be used as a "contract" between Socorro and other systems
+to which we might send our data.
+
+## Changing The Schema
+
+You can change the schema but if you do, **remember to increment the
+`$target_version`**.
+
+The JSON Schema should contain a key called `$target_version`.
+
+* It's important this is an integer that goes up
+* *Don't change this* if you're...
+  * Adding more keys
+  * Editing comments (content of `description` values)
+* *Do change this* if you're...
+  * Changing the type definition of an *existing* key in any way
+  * Add or remove keys from a `required` sub-key
```

Comment:
This is unclear. If I add a new key to the ``json_dump`` object, do I increment the version number?